svgdigitizer/entrypoint.py

```
@@ -571,6 +597,35 @@ def _create_linked_svg(svg, png, svg_template):
         drawing.save(pretty=True)
 
 
+@click.command()
+@click.option(
+    "--template",
+    type=str,
+    default=None,
+    help="Add template elements in svg files. Options: basic, file:<file path>",
+)
+@click.option(
+    "--outdir",
+    type=click.Path(file_okay=False),
+    default=None,
+    help="Write output files to this directory.",
+)
+@click.argument("img")
+def create_svg(img, template, outdir):
+    r"""
+    Write an SVG that shows `png` or `jpeg` as a linked image.
+
+    """
+    import mimetypes
+
+    mimetype = mimetypes.guess_type(img)[0]
+    if mimetype and mimetype.split("/")[1] in ["jpeg", "png"]:
+        svg = _outfile(img, suffix=".svg", outdir=outdir)
+        _create_svg(svg, img, template, True)
+    else:
+        print("Only png and jpeg image formats are supported.")
```

Shouldn't this rather raise an Error?